### PR TITLE
Select Pool UI

### PR DESCRIPTION
**Pull request by antonijzelinskij**
#85ztp5w74
Select pool screen UI

### CODE REVIEW

**Comment by stepanLav** [comment]
Task linked: [CU-85ztp5w74 Android. Start Staking. Select pool screen](https://app.clickup.com/t/85ztp5w74)

File: common/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,11 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="TypographyEllipsis">
+    <string name="select_custom_pool_title">Select pool</string>
+    <string name="select_custom_pool_select_recommedned_chip">Select recommended</string>
```
Comment (valentunn) [inline common/src/main/res/values/strings.xml:4]: This string should already be available afaik. Lets resue existing one

File: feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/presentation/validators/change/ValidatorModel.kt [hunk 2]
```diff
@@ -3,8 +3,11 @@ package io.novafoundation.nova.feature_staking_impl.presentation.validators.chan
 import io.novafoundation.nova.common.address.AddressModel
 import io.novafoundation.nova.common.presentation.ColoredText
 import io.novafoundation.nova.feature_staking_api.domain.model.Validator
+import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.model.NominationPool
 
-typealias ValidatorModel = StakeTargetModel<Validator>
+typealias ValidatorStakeTargetModel = StakeTargetModel<Validator>
+
+typealias PoolStakeTargetModel = StakeTargetModel<NominationPool>
```
Comment (valentunn) [inline feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/presentation/validators/change/ValidatorModel.kt:10]: I suggest to move this typelias definition to nominationPools presentation package

**Review by valentunn — APPROVED** [review]
Can you please attach screenshot of how UI looks like?

File: feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/presentation/validators/change/ValidatorModel.kt (hunk 2 above)
Comment (antonijzelinskij) [reply feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/presentation/validators/change/ValidatorModel.kt:10]: Delete it in logic pool request

**Pull request merged**